README.md

```diff
@@ -28,13 +28,13 @@ fileWatcher.onAdd(awsUploader.onFileAdd);
 Default plugin export. This class is plug-and-play with the Ingest Application Framework, as described in the prevous section.
 
 ### Methods
-`constructor(mediaConvertEndpoint: string, awsRegion: string, ingestBucket: string, outputBucket: string, roleArn:string, playlistName: string, logger: winston.Logger)`
+`constructor(mediaConvertEndpoint: string, awsRegion: string, ingestBucket: string, outputBucket: string, roleArn:string, playlistName: string, encodeParams: string, logger: winston.Logger)`
 
 Creates a new `AWSUploadModule` object. You need to provide the unique part your mediaconvert endpoint URL, which AWS region it is running in, as well as the name of your ingest and output buckets. You will also need to provide a role ARN, as well as the base name of the generated playlist. A winston logger is also needed. These parameters are used to initialize the sub-modules.
 
-`onFileAdd = (filePath: string, readStream: Readable)`. 
+`onFileAdd = (filePath: string, readStream: Readable, fileWatcher: boolean)`.
```

Comment:
When making changes to the interface you also need to update https://github.com/Eyevinn/ingest-application-framework/blob/main/interfaces/interfaces.d.ts and must be general (and not plugin specific)